packages/donations-app/app/components/checkout/amountSelector.html

```diff
@@ -1,69 +1,59 @@
 <div class="row">
```

Comment:
I think it would be cleaner to implement amountSelector using ng-for

Reply:
I have noticed that after the first amount selection some empty input is showing and disappearing in 1 second